### DISCUSSION TITLE
Does the upstash sync work automatically?

### DISCUSSION THREAD

**Discussion by 2p990i9hpral** [ChatGPTNextWeb/NextChat#4676] [Q&A]
Does the upstash sync work automatically? or should I click sync whenever want to update local history to DB?
As a user with multiple devices, manual sync is quite inconvenient. Is that an intended work? Or are there any settings I missed?
I need a tutorial or guide for the proper sync procedure.

**Comment by jiangying000** [comment]
我也有问题：会不会错误地互相覆盖？

有没有类似编辑器那种很复杂的自动合并改动的功能？
